File: src/com/ecci/Hamers/Event.java
package com.ecci.Hamers;

public class Event {

    private String title;
    private String beschrijving;
    private String date;
    private String end_time;

    public Event(String title, String beschrijving, String date, String end_time) {
        super();
        this.title = title;
        this.beschrijving = beschrijving;
        this.date = date;
        this.end_time = end_time;
    }

    public String getTitle() {
        return title;
    }

    public String getBeschrijving() {
        return beschrijving;
    }

    public String getDate() {
        return date;
    }

    public String getEnd_time() {
        return end_time;
    }

}

File: hamersapp/src/main/java/nl/ecci/hamers/events/Event.kt
package nl.ecci.hamers.events

import com.google.gson.annotations.SerializedName
import nl.ecci.hamers.helpers.Utils
import java.util.*

class Event(val id: Int,
            val title: String = Utils.unknown,
            @SerializedName("beschrijving")
            val description: String = Utils.unknown,
            val location: String = Utils.unknown,
            @SerializedName("user_id")
            val userID: Int = 0,
            val date: Date = Date(),
            @SerializedName("end_time")
            val endDate: Date = Date(),
            val deadline: Date = Date(),
            @SerializedName("signups")
            val signUps: ArrayList<SignUp> = ArrayList<SignUp>(),
            @SerializedName("created_at")
            val createdAt: Date = Date(),
            val attendance: Boolean = false) {

    companion object {
        @JvmField val EVENT = "EVENT"
    }

}
